doc/releases/migration-guide-3.5.rst

```diff
@@ -57,6 +57,23 @@ Required changes
   If the Kconfig option is set to ``y`` (current default), then the network
   interface name is used by the ``SO_BINDTODEVICE`` socket option.
 
+* The RAM disk driver has been changed to support multiple instances and instantiation
+  using devicetree. As a result, Kconfig option :kconfig:option:`CONFIG_DISK_RAM_VOLUME_SIZE`
+  and Kconfig option :kconfig:option:`CONFIG_DISK_RAM_VOLUME_NAME` are removed,
+  and the application using the RAM disk must instantiate it using devicetree,
+  as in the following example:
+
+.. code-block:: devicetree
+
+	/ {
+		ramdisk0 {
+			compatible = "zephyr,ram-disk";
+			disk-name = "RAM";
+			sector-size = <512>;
+			sector-count = <192>;
+		};
+	};
```

Comment:
can you please indent this block by 2 spaces so that it's part of the same bullet list?
LGTM otherwise, thx!

Reply:
sorry for the confusion, I meant the actual hihglighted block in my comment, starting at line 66, so that the directive "belongs" to the list item.

Reply:
My fault twice, somehow overlooked exactly that.